osbuild

```diff
@@ -31,9 +31,6 @@ if __name__ == "__main__":
         pipeline = json.load(f)
     pipeline = osbuild.Pipeline(pipeline, args.objects)
 
-    print()
-    print(f"{RESET}{BOLD}Pipeline: {pipeline.id}{RESET}")
-
```

Comment:
Why remove this?

Reply:
Added that to print the pipeline id, but that is no longer relevant. I should remove the whole pipeline id (we only use stage id's now).